src/CacheInvalidator.php

```diff
@@ -321,8 +322,7 @@ public function flush()
 
     private function dispatch(Event $event, $eventName)
     {
-        // LegacyEventDispatcherProxy exists in Symfony >= 4.3
-        if (class_exists(LegacyEventDispatcherProxy::class)) {
+        if ((Kernel::MAJOR_VERSION === 4 && Kernel::MINOR_VERSION >= 3) || Kernel::MAJOR_VERSION >= 5) {
```

Comment:
the kernel is only in the dev dependencies, so we can't access it here.

but as you raise the minimum version of the event dispatcher to 4.3 anyways, i think we can simply drop the check and always do the new way, no?

Reply:
Sure :+1: